moveit_setup_assistant/moveit_setup_controllers/src/moveit_controllers_config.cpp

```diff
@@ -187,6 +187,17 @@ bool MoveItControllersConfig::GeneratedControllersConfig::writeYaml(YAML::Emitte
         emitter << YAML::BeginMap;
         {
           emitter << YAML::Key << "type" << YAML::Value << controller.type_;
+          if (controller.type_ == "FollowJointTrajectory")
+          {
+            emitter << YAML::Key << "action_ns" << YAML::Value << "follow_joint_trajectory";
+          }
+          else
+          {
+            RCLCPP_ERROR_STREAM(*logger_, "Only a FollowJointTrajectory type of controller is supported, for now. "
+                                          "Update moveit_controllers.yaml.");
+            return false;
```

Comment:
this will actually break the ability to re-export if you have other controllers defined/updated (for instance, a gripper controller would cause this to fail to export).

Reply:
Thanks for mentioning that -- I forgot about grippers.

I simplified this to be identical the state before https://github.com/ros-planning/moveit2/commit/8ee767a49912476497a45917057654a3351dc465